components/bases-locales/validator/validateur-documentation/index.js

```diff
@@ -14,15 +14,20 @@ function ValidatorDocumentation() {
         <p>Il est utilisé pour s&apos;assurer avant la publication que toutes les adresses d&apos;une BAL remonteront correctement dans la Base Adresse Nationale.</p>
 
         <span>
-          Il existe deux profils sur le validateur :
+          Il existe quatre profils sur le validateur :
```

Comment:
```suggestion
          Il existe trois profils sur le validateur :
```
Seul trois sont cités, non ?